src/noit_check_log.c

```diff
@@ -234,6 +239,73 @@ noit_check_log_status(noit_check_t *check) {
 }
 
 static int
+account_id_from_name(const char *check_name)
+{
+  int account_id = 0;
+  char *x = strstr(check_name, "`c_");
+  if (x != NULL) {
+    /* take advantage of atoi parsing numbers until the first non-number */
+    account_id = atoi(x + 3);
+  }
+  return account_id;
+}
+
+static int
+noit_check_log_bundle_metric_flatbuffer_serialize_log(mtev_log_stream_t ls,
```

Comment:
right... but the intent behind the other one is clearer, because I can git-grep for the other function name, and see how you intend to call it. I don't know what this function is for...